spec/providers/atmosphere/optimization_strategy/manual_spec.rb

```diff
@@ -11,13 +11,20 @@
   let(:cs) { create(:compute_site, active: true, funds: [fund]) }
   let(:at) { create(:filled_appliance_type, visible_to: 'all', preference_disk: 0) }
   let!(:tmpl) { create(:virtual_machine_template, appliance_type: at, compute_site: cs) }
-  let!(:fl1) { create(:virtual_machine_flavor, cpu: 1, memory: 512, compute_site: cs, active: true, hourly_cost: 1) }
-  let!(:fl2) { create(:virtual_machine_flavor, cpu: 2, memory: 1024, compute_site: cs, active: true, hourly_cost: 2) }
+  let!(:fl1) { create(:virtual_machine_flavor, cpu: 1, memory: 512, compute_site: cs, active: true) }
+  let!(:fl2) { create(:virtual_machine_flavor, cpu: 2, memory: 1024, compute_site: cs, active: true) }
```

Comment:
Please take a look that `fl1` and `fl2` are not used in second test, but these flavors will be created in db. I know that here it is only 2 additional inserts but "ziarnko do ziarnka a zbierze się miarka" :smile: 

Try to refactor it in such a way that every spec will have only required db object created for every spec.